### PR TITLE
master->v10 merge up

### PR DESCRIPTION
**Pull request by lindboe**
## Please verify the following:

- [x] `yarn test` **jest** tests pass with new tests, if relevant
- [x] `yarn lint` **eslint** checks pass with new code, if relevant
- [x] `yarn format:check` **prettier** checks pass with new code, if relevant

### CODE REVIEW

**Review by frankcalise — COMMENTED** [review]
Looking good by eye, will test generating an app shortly.

All I spotted is looks like some binary file snuck in?

File: test/vanilla/.ignite-new.test.ts.swp
[the remark this answers is not in the document]
Comment (lindboe) [reply test/vanilla/.ignite-new.test.ts.swp:1]: Ah yup I had vim crash and it left this file, thanks for catching that

**Comment by lindboe** [comment]
@frankcalise I tested a new app with yarn and CNG, didn't spot any issues

**Pull request merged**